### PR TITLE
Fix of update collection parameters

### PR DESCRIPTION
**Pull request by mrscoopers**

### CODE REVIEW

File: qdrant-landing/content/documentation/concepts/collections.md [hunk 1]
```diff
@@ -927,13 +927,13 @@ client.UpdateCollection(context.Background(), &qdrant.UpdateCollection{
 })
 ```
 
-The following parameters can be updated:
+The following parameters can be updated (parameter names in **python**):
```
[the remark this answers is not in the document]
Comment (mrscoopers) [reply qdrant-landing/content/documentation/concepts/collections.md:930]: The thing which makes me insane - is that parameter names differ in clients
Like for HTTP, we have `params`, but for Python `collection_params`
Tbf the best solution is to list all correct parameter names for each client...

[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/concepts/collections.md:930]: I think we should just stick to how it's named in the REST spec.
https://api.qdrant.tech/v-1-12-x/api-reference/collections/update-collection#request

[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/concepts/collections.md:930]: i.e. Not make any changes.

[the remark this answers is not in the document]
Comment (mrscoopers) [reply qdrant-landing/content/documentation/concepts/collections.md:930]: Yeah, but as a person using Python client, I struggle so much that each time I want to test smth quickly in collab, the easiest way is to download our repo and go to tests, to get all the parameter names right:(

[the remark this answers is not in the document]
Comment (davidmyriel) [reply qdrant-landing/content/documentation/concepts/collections.md:930]: @joein any opinions?

**Comment by davidmyriel** [comment]
to be discussed later

**Pull request closed without merging**